.husky/commit-msg

```diff
@@ -0,0 +1 @@
+npx --no -- commitlint --edit "$1"
```

Comment:
do validation on the commit message?

Reply:
I haven't touched the commitlint config file in this PR.